src/charm.py

```diff
@@ -224,7 +221,7 @@ def _on_install(self, event: InstallEvent) -> None:
                     type="LoadBalancer",
                 ),
             ),
-            field_manager=DEFAULT_FIELD_MANAGER,
+            field_manager=self.__class__.__name__,
```

Comment:
While there's nothing wrong with this, did you consider using `self.app.name` here?
I'm not sure if this will be visible to the user anywhere, but if yes, perhaps pointing to the application name would be slightly better than to the class name?
WDYT?

Reply:
The user could see the manager by using --show-managed-fields option if desired. 
```
gatici@gaticipc:~$ kubectl get service/amf-external -n test1 --show-managed-fields -o yaml 
apiVersion: v1
kind: Service
metadata:
  creationTimestamp: "2024-03-05T19:35:39Z"
  labels:
    app.juju.is/created-by: amf
  managedFields:
  - apiVersion: v1
    fieldsType: FieldsV1
    fieldsV1:
      f:spec:
        f:ports:
          k:{"port":38412,"protocol":"SCTP"}:
            .: {}
            f:name: {}
            f:port: {}
            f:protocol: {}
        f:selector: {}
        f:type: {}
    manager: AMFOperatorCharm
    operation: Apply
    time: "2024-03-05T19:35:39Z"
  - apiVersion: v1
    fieldsType: FieldsV1
    fieldsV1:
      f:status:
        f:loadBalancer:
          f:ingress: {}
    manager: controller
    operation: Update
    subresource: status
    time: "2024-03-05T19:35:39Z"
  name: amf-external
  namespace: test1
  resourceVersion: "16851898"
  uid: 0556b369-0e36-4c01-b278-d87d5dbfc9d6
spec:
  allocateLoadBalancerNodePorts: true
  clusterIP: 10.152.183.100
  clusterIPs:
  - 10.152.183.100
  externalTrafficPolicy: Cluster
  internalTrafficPolicy: Cluster
  ipFamilies:
  - IPv4
  ipFamilyPolicy: SingleStack
  ports:
  - name: ngapp
    nodePort: 31647
    port: 38412
    protocol: SCTP
    targetPort: 38412
  selector:
    app.kubernetes.io/name: amf
  sessionAffinity: None
  type: LoadBalancer
status:
  loadBalancer:
    ingress:
```
Using the app name makes sense to me.

Reply:
Done